frontend/src/views/Create.tsx

```diff
@@ -23,9 +22,7 @@ function CreateGamePage() {
       .then((res) => {
         redirectToLobby(res);
         resolve();
-      }).catch((err) => {
-        reject(errorHandler(err.message));
-      });
+      }).catch((err) => reject(err));
```

Comment:
Is there no need for the `errorHandler(...)` method? If so, should we remove its other instances? (namely in `Socket.ts`, but elsewhere as well)

Reply:
I think the way it's done in `Socket.ts` is correct, but for here I noticed that the REST calls already return `axiosErrorHandler(err)` so I think there's no need to make another call to `errorHandler`.